### PR TITLE
Update CONTRIBUTING doc to mention analysis

### PR DESCRIPTION
**Pull request by Hixie**
cc @leafpetersen - trying to capture what you ran into earlier today.

### CODE REVIEW

**Review by leafpetersen — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -107,15 +128,17 @@ Contributing code
 
 We gladly accept contributions via GitHub pull requests.
 
+Please peruse our
+[style guides](https://flutter.io/style-guide/) and
```
Comment (abarth) [inline CONTRIBUTING.md]: Is this still the correct URL For the style guide?  I thought we moved it to the wiki.

**Review by abarth — APPROVED** [review]
lgtm

File: CONTRIBUTING.md (hunk 1 above)
Comment (Hixie) [inline CONTRIBUTING.md]: Oops, good point.

**Pull request merged**